src/components/Eth2list.js

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import styled from "styled-components"
+
+import Link from "./Link"
+
+const articles = [
+  {
+    title: "What's new in Eth2",
+    description: "Ben Edgington (PegaSys, ConsenSys)",
+    link:
+      "https://hackmd.io/@benjaminion/eth2_news/https%3A%2F%2Fhackmd.io%2F%40benjaminion%2Fwnie2_200725",
+  },
+  {
+    title: "The state of Eth2",
+    description: "Danny Ryan (Ethereum Foundation)",
+    link: "https://blog.ethereum.org/2020/06/02/the-state-of-eth2-june-2020/",
+  },
+  {
+    title: "Eth 2.0 dev updates",
+    description: "Prysmatic Labs",
+    link: "https://medium.com/prysmatic-labs",
+  },
+  {
+    title: "Ethereum.org blog",
+    link: "http://localhost:8000/en/eth2/blog.ethereum.org",
```

Comment:
@ryancreatescopy not sure what this link is supposed to be?

Reply:
Perhaps linking out to the R&D tag would be useful: https://blog.ethereum.org/category/research-and-development/

Reply:
Yeah it was just meant to link to ethereum blog. R&D tag might be useful — shame there's not an eth2 tag!

Reply:
That's a good point, though I believe all eth2 posts fall under the R&D category.

Reply:
fixed, although I think eventual plan is to get rid of this component